roles/fluentbit/tasks/configure.yml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+---
+- name: "Template Fluentbit service file"
+  template:
+    src: fluent-bit.service.j2
+    dest: "/usr/lib/systemd/system/{{ fluentbit_service }}"
+    mode: 0644
+  notify: "systemctl daemon-reload"
+
+- name: "Template Fluentbit configuration"
```

Comment:
We do not render the configuration inside `tdp-collection` with `tdp_vars` like we did for Prometheus exporters ?

Reply:
Pre-req does not know about TDP vars so we render the config in the collection (https://github.com/TOSIT-IO/tdp-collection/pull/652) and they should be loaded "manually" (see example in https://github.com/TOSIT-IO/tdp-getting-started/pull/222).